pkg/kubelet/cpumanager/cpu_manager.go

```diff
@@ -88,6 +94,20 @@ func NewManager(policyName PolicyName, cr internalapi.RuntimeService, kletGetter
 	}, nil
 }
 
+func getReserverdCores(nodeStatus v1.NodeStatus) int64 {
```

Comment:
Mind adding a test? Know we're already in test-debt but this seems like an easy add.